db/src/models/ticket_instances.rs

```diff
@@ -874,11 +883,12 @@ impl TicketInstance {
             }
         }
 
-        if !all_tickets_valid || tickets.len() == 0 {
-            return Err(DatabaseError::new(
-                ErrorCode::BusinessProcessError,
-                Some("User does not own all requested tickets".to_string()),
-            ));
+        if has_redeemed_tickets {
+            return DatabaseError::business_process_error(
+                "Transfer tickets include redeemed tickets not eligible for transferring",
```

Comment:
I was thinking maybe a clearer message?
```suggestion
                "Redeemed tickets cannot be transferred",
```

Reply:
Sounds good -- adjusted this and its test to use this messaging.